spec/controllers/bookings_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'rails_helper'
+require 'auth_helper'
+
+RSpec.describe BookingsController, type: :controller do
+  let(:role) { create(:custom_role) }
+  let!(:user) { FactoryBot.create(:user, role_id: role.id, name: "abc") }
```

Comment:
cant we use directly create(:user, role_id: role.id, name: "abc")?
Why FactoryBot?